content/docs/next-release/opentelemetry.md

```diff
@@ -26,6 +26,7 @@ The Jaeger OpenTelemetry binaries are **almost** backward compatible with the cu
 
 The differences are:
 
+* OTLP receiver listening on port 55680
```

Comment:
Is this the default port in OTEL?  Not a great choice picking a port from ephemeral range, could run into random port conflicts.

Reply:
Yup, 55680 is default.

https://github.com/open-telemetry/opentelemetry-collector/tree/master/receiver/otlpreceiver

I had not thought of that before, but you are correct that could conflict with dynamic ports.

Reply:
I filed https://github.com/open-telemetry/opentelemetry-specification/issues/1148.
Note that Jaeger is using at least one port (6831) assigned to someone else.